advocacy_docs/partner_docs/VisionXGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for VisionX'
```

Comment:
Please review the entire guide and update as needed for the following.
The partner name is SIB Visions and the product is Vision X.  We need to use them appropriately on all pages including the main navigation page.  See past partner guides for examples.